### PR TITLE
Upgrade oc-proto version and fix errors.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/issues/101.

### CODE REVIEW

**Comment by songy23** [comment]
Seems this also depends on https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/pull/48.

File: metrics.go [hunk 1]
```diff
@@ -363,7 +363,7 @@ func labelDescriptorsFromProto(defaults map[string]labelValue, protoLabelKeys []
 }
 
 func metricProseFromProto(metric *metricspb.Metric) (name, description, unit string, ok bool) {
```
Comment (bogdandrutu) [inline metrics.go]: This function name looks like a typo

Comment (odeke-em) [reply metrics.go]: Nah, `prose` https://www.merriam-webster.com/dictionary/prose

**Comment by siriele** [comment]
Is this change going to go live any time soon? It's currently breaking our builds

**Comment by bogdandrutu** [comment]
Hope so, @songy23 are you going to fix this?

**Comment by songy23** [comment]
Yes, as soon as https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/pull/48 is submitted I should be able to submit this PR.

Meanwhile a quick fix is to pin a released version of `opencensus-go-exporter-stackdriver` and use v0.1.0 for `opencensus-proto` in `go.mod`.

**Comment by odeke-em** [comment]
@songy23 please update ocagent-exporter to https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/releases/tag/v0.4.7 in here, do the `GO111MODULE=on go mod tidy` dance afterwards too.

File: go.sum [hunk 2]
```diff
@@ -55,6 +56,7 @@ github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1 h1:EGx4pi6eqNxGa
 github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1/go.mod h1:wJfORRmW1u3UXTncJ5qlYoELFm8eSnnEO6hX4iZ3EWY=
 github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7/go.mod h1:FecbI9+v66THATjSRHfNgh1IVFe/9kFxbXtjV0ctIMA=
 github.com/grpc-ecosystem/grpc-gateway v1.5.0/go.mod h1:RSKVYQBd5MCa4OVpNdGskqpgL2+G+NZTnrVHpWWfpdw=
```
Comment (odeke-em) [inline go.sum]: Did you remember to run `GO111MODULE=on go mod tidy`? This change should have disappeared.

**Comment by odeke-em** [comment]
@songy23 just a few more
>[contrib.go.opencensus.io/exporter/stackdriver.test]
./metrics_test.go:101:5: cannot use promoted field Descriptor in struct literal of type "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1".Metric
./metrics_test.go:234:5: cannot use promoted field Descriptor in struct literal of type "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1".Metric
./metrics_test.go:247:5: cannot use promoted field Descriptor in struct literal of type "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1".Metric
The command "go vet ./..." exited with 2.
$ go test -v -race $PKGS
# contrib.go.opencensus.io/exporter/stackdriver [contrib.go.opencensus.io/exporter/stackdriver.test]
./metrics_test.go:101:5: cannot use promoted field Descriptor in struct literal of type "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1".Metric
./metrics_test.go:234:5: cannot use promoted field Descriptor in struct literal of type "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1".Metric
./metrics_test.go:247:5: cannot use promoted field Descriptor in struct literal of type "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1".Metric

keep going champ!

File: go.sum (hunk 2 above)
Comment (songy23) [reply go.sum]: Yes, I guess there're other dependencies that depend on it?

**Comment by songy23** [comment]
> @songy23 just a few more

Thanks, that was fixed. I'll submit it once CI is green. 

A price paid for making breaking changes...

**Pull request merged**